### PR TITLE
Feature/wteg feedback

### PR DESCRIPTION
**Pull request by JamesWChan**
Hi @punalsc,

Would it be possible to have a look at this PR?

Thanks!

### CODE REVIEW

File: components/feedback/child_components/form/wteg_feedback_form/wteg_feedback_form.react.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+import Fieldset from '../../fieldset/details_feedback_fieldset.react';
+import Comment from "../../comment/details_feedback_comment.react";
+import { Data } from '../../../data';
+import Message from "../../message/details_feedback_message.react";
+
+class WTEGFeedbackForm extends Component{
+	constructor(props){
+		super(props);
+
+		this.buildGtmObj = this.buildGtmObj.bind(this);
```
Comment (punalsc) [inline components/feedback/child_components/form/wteg_feedback_form/wteg_feedback_form.react.js]: Could we use arrow functions for the methods? Then we will not need to bind them, as arrow functions will have its own scope.

File: components/feedback/child_components/form/wteg_feedback_form/wteg_feedback_form.react.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+		this.handleChange = this.handleChange.bind(this);
+		this.handleSubmit = this.handleSubmit.bind(this);
+		this.pushInDataLayer = this.pushInDataLayer.bind(this);
```
Comment (punalsc) [inline components/feedback/child_components/form/wteg_feedback_form/wteg_feedback_form.react.js]: this is not needed.

File: components/feedback/style.scss [hunk 3]
```diff
@@ -39,7 +39,7 @@ $tna_outline: #fce45c;
   }
 }
 
-#details-feedback-wrapper {
+#details-feedback-wrapper, #what-to-expect {
```
Comment (punalsc) [inline components/feedback/style.scss]: can have the id as `#what-to-expect-form`

File: components/feedback/wteg_feedback_widget.react.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import WTEGFeedbackForm from './child_components/form/wteg_feedback_form/wteg_feedback_form.react';
+
+import './style.scss';
+
+class WTEGFeedbackWidget extends Component {
+	render() {
+		return (
+			<React.Fragment>
+				<h2>Feedback</h2>
+				<WTEGFeedbackForm />
```
Comment (punalsc) [inline components/feedback/wteg_feedback_widget.react.js]: As discussed, give the a `prop` eg: `<WTEGFeedbackForm id='what-to-expect-form' />`

File: src/runners/wteg-feedback-runner.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import WTEGFeedbackWidget from '../../components/feedback/wteg_feedback_widget.react';
+
+{
+	const detailsPage = document.querySelector('.details-page'),
+		path = window.location.pathname,
+		url = window.location.href,
+		wtegWrapperId = 'what-to-expect',
```
Comment (punalsc) [inline src/runners/wteg-feedback-runner.js]: Please make sure it's changed to `what-to-expect-form`

File: components/feedback/child_components/form/wteg_feedback_form/wteg_feedback_form.react.js (hunk 1 above)
Comment (JamesWChan) [reply components/feedback/child_components/form/wteg_feedback_form/wteg_feedback_form.react.js]: Done

File: components/feedback/style.scss (hunk 3 above)
Comment (JamesWChan) [reply components/feedback/style.scss]: Done

File: components/feedback/wteg_feedback_widget.react.js (hunk 4 above)
Comment (JamesWChan) [reply components/feedback/wteg_feedback_widget.react.js]: Done

File: src/runners/wteg-feedback-runner.js (hunk 5 above)
Comment (JamesWChan) [reply src/runners/wteg-feedback-runner.js]: Done

**Pull request merged**